runtime/doc/filetype.txt

```diff
@@ -829,6 +829,30 @@ The mappings can be disabled with: >
 	let g:no_vim_maps = 1
 
 
+ZIG							*ft-zig-plugin*
+
+						*g:zig_recommended_style*
+By default the following indentation options are set, in accordance with Zig's
+recommended style (https://ziglang.org/documentation/master/):
+
+	`setlocal expandtab shiftwidth=4 softtabstop=4 tabstop=8`
+
+To disable this behavior, set g:zig_recommended_style to 0:
+
+	`let g:zig_recommended_style = 0`
+
+						*g:zig_std_dir*
+The path to the zig standard library. The Zig |ftplugin| reads g:zig_std_dir
+and appends it to the |path| for zig files. Where the zig standard library
+is located is system and installation method dependant.
+
+One can automatically read the std_dir using `zig env`:
+	`let g:zig_std_dir = json_decode(system('zig env'))['std_dir']`
+
+This can, for example, be put in a FileType |autocmd| to only load when a zig
+file is opened.
```

Comment:
```suggestion
One can automatically set |g:zig_std_dir| using `zig env`: >
	let g:zig_std_dir = json_decode(system('zig env'))['std_dir']
<
This can, for example, be put in a FileType |autocmd| or user |ftplugin| to only load when a Zig
file is opened.
```